common/source/docs/common-autopilots.rst

```diff
@@ -30,6 +30,7 @@ Open hardware
     Holybro Pixhawk 4 <https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL_ChibiOS/hwdef/Pixhawk4/README.md>
     Holybro Pixhawk6X <common-holybro-pixhawk6X>
     Holybro Pixhawk6C <common-holybro-pixhawk6C>
+    Holybro Pix32v6(Pixhawk6 variant) <common-holybro-pix32v6>
```

Comment:
Please change to 
`Holybro Pix32v6 (Pixhawk6C variant) <common-holybro-pix32v6>`